.github/workflows/ci.yml

```diff
@@ -57,14 +57,23 @@ jobs:
         run: |
           go build -o runme main.go
           ./runme --version
+      - name: Test All
+        run: |
+          export SHELL=/bin/bash
+          export TZ=UTC
+          make test
+        if: ${{ matrix.os == 'ubuntu-latest' }}
       - name: Test
         run: |
           export SHELL=/bin/bash
           export TZ=UTC
           TAGS="test_with_docker" make test/coverage
           make test/coverage/func
         if: ${{ matrix.os == 'ubuntu-latest' }}
+      # make test should run on all os's because
+      # on windows only a subset of the tests run.
       - name: Test
+        # N.B. Set-Timezone is a windows command
```

Comment:
I don't understand this comment. It is set before calling `make test`: `Set-Timezone -Id "UTC" -PassThru`.

Reply:
I believe it's just a note to make clear that `Set-Timezone` is a Windows PowerShell command; for folks who are not familiar with Windows details. Please confirm @jlewi.

Reply:
That's right. Happy to remove it if you prefer less comments.